README.md

```diff
@@ -1 +1,3 @@
-# ea_restaurant_data_loader
\ No newline at end of file
+# ea_restaurant_data_loader
+
+ea_restaurant_data_loader is a microservice as part of [ea_restaurant](https://github.com/eapg/EA_RESTAURANT) project to load data into database from an excel file using the functional programming language elixir.
```

Comment:
I would suggest adding more information, try to answer these questions:
```
What is this project about?
Why do we need this project for?
How will it impact the whole application?
Do you have any examples on how it will impact the application?
```

Reply:
Please check it again.